### PR TITLE
Feature(Header): Add multiline ellipsis support

### PR DESCRIPTION
**Pull request by BazanAlexandro**
*Implemented multiline ellipsis via CSS. The only way to do it is via line-clamp prop which is working only in these browsers https://caniuse.com/#feat=css-line-clamp
If browser doesn't support line-clamp, the fallback solution will be applied: text will be just cropped*

#### Please tick a box ###
- [x] **feature** _(A new feature_)

#### If you're adding a feature, is it documented in a storybook story?
- [x] Yes

#### Are the components you're working on reusable between brands?
- [x] Yes _(Using variables that are defined in the default theme)_

#### If you're creating markup, did you add proper semantics? 
- [x] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/Heading.js [hunk 1]
```diff
@@ -54,12 +63,14 @@ ProtoHeading.propTypes = {
 	marginBottomFactor: propTypes.number,
 	defaultSize: propTypes.string,
 	upSize: propTypes.string,
+	maxLines: propTypes.number,
```
Comment (sergeyvlakh) [inline src/atoms/Heading.js]: but default props in null which is not a number?

File: src/atoms/Heading.js [hunk 2]
```diff
@@ -54,12 +63,14 @@ ProtoHeading.propTypes = {
...
 };
 ProtoHeading.defaultProps = {
 	marginTopFactor: 3/4,
 	marginBottomFactor: 1/2,
 	defaultSize: 'Regular',
 	upSize: 'Large',
+	maxLines: null,
```
Comment (misund) [inline src/atoms/Heading.js]:
```suggestion
	maxLines: 0,
```

File: src/atoms/Heading.js [hunk 3]
```diff
@@ -35,6 +35,15 @@ const ProtoHeading = styled.h1`
 					calc(${props.marginBottomFactor} * ${headingDefaultLineHeight})
 				;
 
+				${props.maxLines && `
+					display: -webkit-box;
```
Comment (misund) [inline src/atoms/Heading.js]: If we don't make it backwards-compatible, won't our build systems add browser prefixes and such that is needed?

Comment (BazanAlexandro) [reply src/atoms/Heading.js]: yeah, but we don't have the ability to multiline CSS
if we're using Firefox, text will be just cropped. That's all
in Chrome, we're getting additional ...
However, I could implement JS solution, but it requires time

Comment (misund) [reply src/atoms/Heading.js]: CSS > JS

Comment (misund) [reply src/atoms/Heading.js]: Could you add this explanation to the PR description, though?

Comment (BazanAlexandro) [reply src/atoms/Heading.js]: Sure, doing it

**Comment by misund** [comment]
Percy indicates we missed something

**Comment by Tehtehteh** [comment]
> Percy indicates we missed something
> something
> 88 visual changes
😅

**Comment by BazanAlexandro** [comment]
Semicolon was missing in one of styles. Fixed

**Pull request merged**